wurst/utils/TextTagUtils.wurst

```diff
@@ -0,0 +1,35 @@
+package TextTagUtils
+
+// Local imports:
+import PlayerExtensions
+
+
+/**
+    This function create a textTag upon the target unit and make it fade away upward
+    Visible only by allied player
+**/
+public function createTeamFadingTextTag(unit target, string text, color color) returns texttag
+    // Create a text tag to visualize the effect.
+    if localPlayer.canSee(target)
+        return CreateTextTag()
```

Comment:
Why don't you forward the call to the below function? This seems like unnecessarily duplicated code unless you expect the behavior like velocity to not always be the same (which seems doubtful).